src/eShop.AppHost/Program.cs

```diff
@@ -15,12 +15,11 @@
 var orderDb = postgres.AddDatabase("orderingdb");
 var webhooksDb = postgres.AddDatabase("webhooksdb");
 
-var openAi = builder.AddAzureOpenAI("openai");
+var openAi = builder.AddConnectionString("openai");
```

Comment:
Why not AddAzureOpenAIClient?

Reply:
I am curious if this will provision an Open AI resource if you provision with azd as well.

Reply:
I am curious if this will provision an Open AI resource if you provision with azd as well.

Reply:
> I am curious if this will provision an Open AI resource if you provision with azd as well.

it will.

Reply:
I've added code for the following options:

* No openAI (the default)
* Set `<UseAzureOpenAI>true</UseAzureOpenAI>` in eShop.AppHost and set up a SubscriptionId and Location - it will provision an AOAI resource for you when you run
* Set `<UseAzureOpenAI>true</UseAzureOpenAI>` and `ConnectionStrings:openai` in eShop.AppHost to an existing OpenAI instance, and it will be used